x/auctions/keeper/auction.go

```diff
@@ -0,0 +1,23 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/evmos/evmos/v18/x/auctions/types"
+)
+
+// GetRound gets the current auction round
+func (k *Keeper) GetRound(ctx sdk.Context) uint64 {
+	store := ctx.KVStore(k.storeKey)
+	bz := store.Get(types.KeyPrefixAuctionRound)
+	if bz == nil {
+		return 0
+	}
+	round := sdk.BigEndianToUint64(bz)
+	return round
+}
+
+// SetRound sets the current auction round
+func (k *Keeper) SetRound(ctx sdk.Context, round uint64) {
```

Comment:
Setter method can be private right?

Reply:
Not quite, it's used in the hook from the keeper so it should stay public.